api/xds_protocol.rst

```diff
@@ -267,6 +269,14 @@ Later, an API update may succeed at a new version **Y**:
 .. figure:: diagrams/later-ack.svg
    :alt: ACK after NACK
 
+ACK and NACK semantics can be summarized as follows:
+
+- the xDS client should ACK or NACK every *DiscoveryResponse* received from the management server.
+- nonce from the *DiscoveryResponse* is included as response_nonce in the ACK or NACK.
```

Comment:
The `nonce` is not only for ACK/NACK; it actually gets sent in all requests except for the very first one on the stream for each resource type.  I suggest adding a link down to the "Resource updates" section which explains the use of `nonce` in more detail.

Reply:
Yes, didn't mean to imply that nonce is only in ACK/NACK. Will rephrase it (and add the link you suggested) to clarify. But the nonce *is* used to differentiate between a NACK and a stale ACK in certain cases of race conditions as described elsewhere.